ConfigurationSystem/private/ServiceInterface.py

```diff
@@ -109,7 +114,96 @@ def __checkSlavesStatus(self, forceWriteConfiguration=False):
                                                 ", ".join(self.dAliveSlaveServers.keys())))
       self.__generateNewVersion()
 
-  def updateConfiguration(self, sBuffer, commiter="", updateVersionOption=False):
+  @staticmethod
+  def __forceServiceUpdate(url, fromMaster):
+    """
+    Force updating configuration on a given service
+
+    :param str url: service URL
+    :param bool fromMaster: flag to force updating from the master CS
+    :return: S_OK/S_ERROR
+    """
+    if url.startswith('dip'):
+      rpc = RPCClient(url)
+      gLogger.info('Updating service configuration on', url)
+      result = rpc.refreshConfiguration(fromMaster)
+    elif url.startswith('http'):
+      hostCertTuple = getHostCertificateAndKeyLocation()
+      resultRequest = requests.get(url,
+                                   headers={'X-RefreshConfiguration:': "True"},
```

Comment:
Shouldn't this be `fromMaster` instead of `True`? And for my curiosity, what is the meaning of the `X-` ?

Reply:
This is how Andrii implemented it in the WebApp and OAuth http handlers.